src/Symfony/Component/PropertyAccess/Tests/Mapping/ClassMetadataTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Tests\Mapping;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\PropertyAccess\Mapping\ClassMetadata;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class ClassMetadataTest extends TestCase
+{
+    public function testInterface()
+    {
+        $classMetadata = new ClassMetadata('name');
+        $this->assertInstanceOf('Symfony\Component\PropertyAccess\Mapping\ClassMetadata', $classMetadata);
+    }
+
+    public function testAttributeMetadata()
+    {
+        $classMetadata = new ClassMetadata('c');
+
+        $a1 = $this->createMock('Symfony\Component\PropertyAccess\Mapping\PropertyMetadata');
```

Comment:
This seems to fail on PHP 5.5 & PHPUnit 4.8.35. `ClassMetadataTest` extends `PHPUnit\Framework\TestCase` which implements that method, so I'm not sure on how to solve this. It doesn't fail on any other scenarios. Any ideas?